### PR TITLE
jq 1.6

### PR DESCRIPTION
**Pull request by zmwangx**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/jq.rb [hunk 1]
```diff
@@ -13,18 +17,10 @@ class Jq < Formula
...
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool" => :build
-  end
-
   depends_on "oniguruma" # jq depends > 1.5
 
   def install
-    system "autoreconf", "-iv" if build.head?
+    system "autoreconf", "-iv"
```
[the remark this answers is not in the document]
Comment (fxcoudert) [reply Formula/jq.rb]: Can we file an issue about that, so it is fixed for future releases?

[the remark this answers is not in the document]
Comment (zmwangx) [reply Formula/jq.rb]: Sure. https://github.com/stedolan/jq/issues/1352#issuecomment-435390228

[the remark this answers is not in the document]
Comment (zmwangx) [reply Formula/jq.rb]: Switching to the auto-generated tarball for now, in case the tarball is updated.

[the remark this answers is not in the document]
Comment (zmwangx) [reply Formula/jq.rb]: Problem gone.

**Comment by fxcoudert** [comment]
Thanks @zmwangx

**Pull request closed without merging**